src/routes/version_file.rs

```diff
@@ -357,14 +355,12 @@ pub async fn get_versions_from_hashes(
     .fetch_all(&**pool)
     .await?;
 
-    let versions_data = database::models::Version::get_many_full(
-        result
-            .iter()
-            .map(|x| database::models::VersionId(x.version_id))
-            .collect(),
-        &**pool,
-    )
-    .await?;
+    let version_ids = result
+        .iter()
+        .map(|x| database::models::VersionId(x.version_id))
+        .collect::<Vec<_>>();
```

Comment:
If being explicit elsewhere, why not be explicit here? The type inferred is already `Vec<VersionId>`